packages/amplify-graphql-schema-generator/src/certs/cert.ts

```diff
@@ -0,0 +1,33 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+// This logic to get the SSL config must match with the SQL Lambda Layer.
+// If you are changing this, please make sure to change the logic in the SQL Lambda Layer as well.
+export const getSSLConfig = (
```

Comment:
This logic is copied from lambda layer. For now, it is duplicated. We can think about storing this in one place in the future.